docs/tutorials/configure/pet-photographer.md

```diff
@@ -16,6 +16,12 @@ cost: "0"
 no_list: true
 ---
 
+<!-- LEARNING GOALS
+After following this tutorial, you will understand how to control sync parameters, so you can build filters that allow you to filter the data you are collecting from your machines.
```

Comment:
Nit: to avoid `so`:

```suggestion
After following this tutorial, you will understand how to control sync parameters and build filters that allow you to filter the data you are collecting from your machines.
```